packages/shared/components/atoms/pills/ActivityTypePill.svelte

```diff
@@ -0,0 +1,68 @@
+<script lang="typescript">
+    import { localize } from '@core/i18n'
+    import { isParticipationPayload } from '@lib/wallet'
+    import Pill from './Pill.svelte'
+
+    export let activity
+    export let confirmed
+
+    enum ActivityType {
```

Comment:
I think `ActivityType` isn't quite descriptive enough, maybe `TransactionActivityType` or something?

Reply:
But a transaction is a type of activity, in the future when we have more types of activities that transactions :)